Documentation/community/roadmap.rst

```diff
@@ -171,15 +171,18 @@ production.
 Codebase modularization
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-As the project is growing in complexity it is becoming increasingly important to be able
-to divide it into more manageable chunks. To achieve this, we're working on modularizing the
-codebase and going from a tightly coupled design (one large initialization and configuration)
-to a more loosely coupled design of mostly self-contained modules. This will make Cilium 
-internals easier to comprehend, test and extend. 
-
-Contributions in this area are very welcome. To get started, take a look at the :ref:`guide-to-the-hive`
-documentation and the issues referenced from `modularization meta issue <modularization-issue_>`_.
-If you have any questions or ideas please join us on the #sig-modularization channel on `Cilium Slack <slack_>`_.
+As the project is growing in complexity it is becoming increasingly important
+to be able to divide it into more manageable chunks. To achieve this, we're
+working on modularizing the codebase and going from a tightly coupled design
+(one large initialization and configuration) to a more loosely coupled design
+of mostly self-contained modules. This will make Cilium internals easier to
+comprehend, test and extend.
+
+Contributions in this area are very welcome. To get started, take a look at the
+:ref:`guide-to-the-hive` documentation and the issues referenced from
+`modularization meta issue <modularization-issue_>`_. If you have any questions
+or ideas please join us on the ``#sig-modularization`` channel on `Cilium
```

Comment:
Use plain, clear language (avoid "please")
```suggestion
or ideas, join us on the ``#sig-modularization`` channel on `Cilium
```

Reply:
Not part of this PR.